git/essentials/commits/what-is-stored-in-a-commit.md

```diff
@@ -39,3 +40,17 @@ The usual commit object contains a hash-code, author’s name and email, date an
 - description
 - SHA-1
 - checksum
+
+---
+
+## Footnotes
+
+[1: Checksum]
+
+Checksums are used to check files and other data to confirm they are identical.
+
+A checksum is produced by running the file through an algorithm (e.g. MD5, SHA-1, SHA-256, SHA-512) and it produces a string of fixed length. The size or type of the file doesn't matter, the length of the string will always be the same.
```

Comment:
```suggestion
A checksum is produced by running the file through an algorithm (e.g. MD5, SHA-1, SHA-256, SHA-512) and it produces a string of fixed length. The size or type of the file doesn't matter and the length of the string will always be the same.
```

Reply:
Should we add an example of a checksum maybe? We can generate one for an image file we host?

Reply:
good shout, will add an example :+1: